tests/tests.js

```diff
@@ -1,33 +1,157 @@
+} = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
+chai.use(sinonChai)
+const {
+  expect
+} = chai
+
+describe('formatPrice', () => {
+  let spyOnPrice
+  let spyOnformatPrice
+  let sandbox
+
+  foreEach(() => {
+    sandbox = sinon.createSandbox()
+
+    priceSpy = sandbox.spy(pricing, 'price')
+    formatPriceSpy = sandbox.spy(pricing, 'formatPrice')
+  })
+  afterEach(() => {
+    sandbox.restore()
+    it('returns a truncated price with two decimal places not rounded when given a number with multiple places', () => {
+      const x = 15.1566663
+      const formattedPrice = pricing.formatPrice(15.1566663)
+
+      expect(formattedPrice).to.equal(15.15)
+    })
+  })
+
+  describe('calculatesLTDPrice', () => {
+    it('returns the price for a disability product for an employee', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee']
+      }
+      const price = pricing.calculateLTDPrice(product.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+    })
+  })
+
+  it('returns the LTD price minus the employer contribution', () => {
     const selectedOptions = {
-      familyMembersToCover: ['ee'],
-      coverageLevel: [{ role: 'ee', coverage: 125000 }],
+      familyMembersToCover: [ee]
     }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+    const price = pricing.calculateLTDPrice(products.voluntaryLife, employee, selectedOptions)
+  })
+
+  describe('calculatesCommuterPrice', () => {
+    it('returns price for train', () => {
+      const selectedOptions = {
+        benefit: 'train'
+      }
+      const price = pricing.calculateCommuterPrice(products.commuter, selectedOptions)
+      expect(price).to.equal(84.75)
+    })
+    it('returns the price for parking', () => {
+      const selectedOptions = {
+        benefit: 'parking'
+      }
+      const price = pricing.calculateCommuterPrice(products.commuter, selectedOptions)
+      expect(price).to.equal(250)
+    })
+  })
 
-    expect(price).to.equal(39.37)
+  describe('getEmployerContribution', () => {
+    it('returns the employer contribution when given in dollars', () => {
+      const employerContribution = {
+        mode: 'percentage',
+        contribution: 10
+      }
+      expect(pricing.getEmployerContribution(employerContribution, 300)).to.equal(100)
+    })
   })
+  let employerContribution
+  let price
+  let dollarsOff
 
+  foreEach(() => {
+    sandbox = sinon.createSandbox()
+
+    getEmployerContributionSpy = sandbox.spy(pricing, 'getEmployerContribution')
+    priceSpy = sandbox.spy(pricing, 'price')
+    dollarsOff = sandbox.spy(price * (employerContribution.contribution / 100))
+  })
+  afterEach(() => {
+    sandbox.restore()
+  })
```

Comment:
All of this code is duplicative of what you're doing in your `calculateProductPrice` describe